pysilcam/silcamgui/liveviewer.py

```diff
@@ -158,20 +161,20 @@ def liveview(datapath = '/mnt/DATA/emlynd/DATA/', config_filename = 'config_hard
             im = convert_image(imcrop, size)
         else:
             im = convert_image(imraw, size)
-            label = font.render('ZOOM [F]: OFF', 1, (255, 255, 0))
+            label = font.render('ZOOM [F]: OFF', 1, font_colour)
 
         screen.blit(im, (0, 0))
         screen.blit(label,(0, size[1]-20))
 
-        label = font.render('pause[p] write[scpace] exit[Esc]', 1, (255,255,0))
+        label = font.render('pause[p] write[space] exit[Esc]', 1, font_colour)
         screen.blit(label, (0, size[1]-40))
 
         pygame.display.set_caption('Image display')
         label = font.render(str(timestamp) + '    Disp. FPS: ' +
-                str(c.get_fps()), 20, (255, 255, 0))
+                str(c.get_fps()), 20, font_colour)
```

Comment:
E128: continuation line under-indented for visual indent  <!-- Automatically posted by Lintly -->